src/pages/certification/components/AuditorRunTestForm/AuditorRunTestForm.tsx

```diff
@@ -167,7 +168,7 @@ const AuditorRunTestForm: React.FC = () => {
   };
 
   const formHandler = (formData: TestingForm) => {
-    dispatch(createTestRun({}));
+    dispatch(createTestRun(impersonate ? retainId : null));
```

Comment:
Maybe we should check the `impersonate` state inside the `createTestRun` action, so we can avoid checking and sending this value inside this component

Reply:
Like we do in this [wallet connection action](https://github.com/input-output-hk/dapps-certification-web/blob/master/src/store/slices/walletConnection.slice.ts#L114)